index.js

```diff
@@ -121,7 +121,7 @@ types[JCB] = {
   prefixPattern: /^(2|21|213|2131|1|18|180|1800|3|35)$/,
   exactPattern: /^(2131|1800|35)\d*$/,
   gaps: [4, 8, 12],
-  lengths: [16],
+  lengths: [16, 17, 18, 19],
```

Comment:
Our contact at Discover confirmed that JCB cards can be 19 digits. This should be changed to [16, 19].

Reply:
Our contact with Discover is now giving us conflicting information and saying that it _should_ only be 16 and 19. Trying to get a definitive answer for this.

Reply:
Thanks. Keep us posted.

Reply:
Okay, finally got a confirmation. Currently, only 16 or 19 digit cards have been issued, but issuers are allowed to issue 17 and 18 digit cards, so the logic you have is what we should go with for now. 

Should be able to get this merged soon.

Reply:
Good to hear!